src/app/modules/shared-components/components/small-section/small-section.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="progress-section-container">
```

Comment:
The name `small-section` does not really reflect what the component does, can you find a more precise name ?
Some ideas to narrow the search: `form-control-section`, `control-section`

Reply:
fixed, it has been renamed `collapsible-section`